// [[Reason: Semantically invalid. TSC does not emit .d.ts]] ////

//// [tests/cases/compiler/declarationEmitLambdaWithMissingTypeParameterNoCrash.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,5 +1,12 @@
 
+
+//// [declarationEmitLambdaWithMissingTypeParameterNoCrash.d.ts]
+export interface Foo {
+    preFetch: <T1 extends T2>(c: T1) => void;
+    preFetcher: new <T1 extends T2>(c: T1) => void;
+}
+//# sourceMappingURL=declarationEmitLambdaWithMissingTypeParameterNoCrash.d.ts.map
 /// [Errors] ////
 
 declarationEmitLambdaWithMissingTypeParameterNoCrash.ts(2,27): error TS2304: Cannot find name 'T2'.
 declarationEmitLambdaWithMissingTypeParameterNoCrash.ts(2,27): error TS4016: Type parameter 'T1' of exported function has or is using private name 'T2'.
